cli/args/flags.rs

```diff
@@ -2087,7 +2088,8 @@ fn v8_flags_arg<'a>() -> Arg<'a> {
     .use_value_delimiter(true)
     .require_equals(true)
     .help("Set V8 command line options")
-    .long_help("To see a list of all available flags use --v8-flags=--help.")
+    .long_help("To see a list of all available flags use --v8-flags=--help.\
+    Any flags set with this flag are appended after the DENO_V8_FLAGS environmental variable")
```

Comment:
TIL that unlike JS, the backslash on the previous line causes the indentation on this line to be ignored.